### DISCUSSION TITLE
XInclude or other include mechanism in the scene XML?

### DISCUSSION THREAD

**Discussion by buddhasystem** [mitsuba-renderer/mitsuba3#1241] [Q&A] [answered]
Are there ways to include other XML files into the scene XML? I've done similar operations in a different project, but I can't make Mitsuba parse XInclude, for example. Has this been done? Thing is, there is sometimes a lot of boilerplate XML shared across a few scenes that should ideally be edited in one place. Hence, the question... Thank you!

**Comment by merlinND** [comment, marked as answer]
Hello @buddhasystem,

See the `<include>` tag:
https://mitsuba.readthedocs.io/en/stable/src/key_topics/scene_format.html#including-external-files

**Reply by buddhasystem** [reply]
Fantastic! It did work. Sorry I missed that in the first reading. Great.
